amethyst_rendy/src/submodules/material.rs

```diff
@@ -199,6 +199,7 @@ impl<B: Backend, T: for<'a> StaticTextureSet<'a>> MaterialSub<B, T> {
     }
 
     /// Attempts to insert a new material to this collection.
+    #[allow(clippy::block_in_if_condition_stmt)]
```

Comment:
Why not fix this with a `let` instead of allowing the lint?

Reply:
This is still not resolved. The lint is correct in that it's annoying to read.

Reply:
I didn't feel like doing it for this first pass.